### PR TITLE
update and fix bug

### PR DESCRIPTION
**Pull request by banguyenvareal**

### CODE REVIEW

File: app/assets/javascripts/request_images.js [hunk 1]
```diff
@@ -2,6 +2,10 @@ $(function(){
   $('.small-img-style').each(function(){
     $(this).click(function(){
       $('#main_image').attr('src', $(this).attr('src'));
+      $('.small-img-style').css('border-width', '0px');
```
Comment (dinhvareal) [inline app/assets/javascripts/request_images.js]: add to class and use `.addClass()` in `jQuery`

File: app/controllers/book_requests_controller.rb [hunk 2]
```diff
@@ -28,6 +29,42 @@ def show
...
+      flash[:success] = t('book_requests.update.success')
+      redirect_to book_request_path(@book_request)
+    else
+      render 'edit'
+    end
+  end
+
+  def destroy
+    path = request.referrer
+    # binding.pry
```
Comment (dinhvareal) [inline app/controllers/book_requests_controller.rb]: remove

File: app/views/book_requests/_list.html.erb [hunk 3]
```diff
@@ -2,21 +2,26 @@
...
     <div class="col-md-3 book-request">
       <div class="row">
-        <img src="<%= book_request.book_request_images.first&.thumb_url %>
+        <img src="<%= get_thumb_url(book_request) %>
           "alt = "Book's cover" class="book-request-image">
         <div class="col-md-12 book-request-content" >
           <p class="book-request-name"> <%= truncate(book_request.name, length: 25) %></p>
           <p class="book-request-detail"><%=t('book_request.buyer') %>
-            <%= book_request.buyer_name %></p>
+            : <%= book_request.buyer_name %></p>
```
Comment (dinhvareal) [inline app/views/book_requests/_list.html.erb]: let it in a line

File: app/views/book_requests/_validate.js [hunk 4]
```diff
@@ -0,0 +1,28 @@
+
```
Comment (dinhvareal) [inline app/views/book_requests/_validate.js]: remove blank line

File: app/views/sessions/_validate.js [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+    $(element).addClass('border-red');
+  },
+  unhighlight: function(element) {
+    $(element).removeClass('border-red');
+  }
+});
+
+$.validator.addMethod("valid_phone", function(value, element) {
+  return this.optional(element) || /[0]\d{9}/i.test(value);
+}, "Phone include 10 digest and begin by 0");
```
Comment (dinhvareal) [inline app/views/sessions/_validate.js]: Insert a blank line

File: app/views/users/book_requests/index.html.erb [hunk 6]
```diff
@@ -1,6 +1,6 @@
 <div class="row">
   <div class="col-sm-10 col-sm-offset-1">
-    <%= render partial: 'list', locals: { book_requests: @book_requests } %>
+    <%= render partial: 'book_requests/list', locals: { book_requests: @book_requests } %>
```
Comment (dinhvareal) [inline app/views/users/book_requests/index.html.erb:3]: `<%= render partial: 'list', locals: { book_requests: @book_requests } %>`

File: spec/controllers/book_requests/destroy_spec.rb [hunk 7]
```diff
@@ -61,15 +65,15 @@
...
+          expect(subject).to redirect_to(request.referrer)
         end
       end
     end
 
     context 'book request is not found' do
       before do
         delete :destroy, params: {
-          user_id: current_user.id, id: BookRequest.last.id + 10, page: page
+          id: BookRequest.last.id + 10, page: page
```
Comment (dinhvareal) [inline spec/controllers/book_requests/destroy_spec.rb]: use stubs

**Review by dinhvareal — APPROVED** [review]

**Review by hieunguyentrung — APPROVED** [review]

**Pull request merged**